hack/run-test-serial-e2e.sh

```diff
@@ -197,12 +197,16 @@ function runtests() {
 }
 
 # Make sure that we always properly clean the environment
-trap 'teardown' EXIT SIGINT SIGTERM SIGSTOP
+trap 'teardown' EXIT SIGINT SIGTERM
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2173

Reply:
this was better done in a separate commit for documentation purposes. Reason is git history is always avaialble and should be self-sufficient. Otherwise, we now need to access github and review the PR review history to learn the why for this change.

Not a big deal but let's try to move towards this direction in the future

Reply:
You're right I was lazy. fixed that